### ISSUE TITLE
when accessing some sites of Brazilian banks the keys of acceleration of nvda stop working

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9339]
### Steps to reproduce:
by accessing the websites of some Brazilian banks such as https://www.itau.com.br/ or https://internetbanking.caixa.gov.br/
The nvda keys like h for headers, e for editable fields and all the others stop working.
Nvda also stops announcing the typed characters.
I believe that the reason is a software name Warsaw 2.8.0.61 64 bits; GAS Technology that banks require the facility to allow access to the bank's website.
But this is an axism without a technical background.
to reproduce the problem:
go to:
https://www.itau.com.br/
Now try to type some acceleration keys like e for edit box, l for 
### Actual behavior:
these acceleration keys do not work, and nvda does not announce the keystrokes.
### Expected behavior:
the acceleration keys like e for edit fields, l for lists should work correctly, and nvda should continue announcing the keystrokes.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
alfa 16874
#### Windows version:
10 17763.316
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
by looking at the nvda log, I see that the letter and number keys typed are not identified by the nvda.

**Comment by rhcarvalho** [comment]
> I believe that the reason is a software name Warsaw 2.8.0.61 64 bits; GAS Technology that banks require the facility to allow access to the bank's website.

I too believe that is very likely to be the cause of the problem, and nothing could be done from NVDA to solve it.

Did you try reporting the problem to the banks?

**Comment by Adriani90** [comment]
@fernando-jose-silva I cannot reproduce this issue. Can you please tell us which browser you are using? And please Elaborate a bit more on the Software which you think is causing this issue. Is that Software installed on your PC?

**Comment by fernando-jose-silva** [comment]
Thank you
I'm using chrome 72.
In Brazil I do not know if in the rest of the world when accessing a website of a Brazilian bank the site shows a message informing that to access this site and to have access to its current account it is necessary to install a software in the computer.
So we downloaded this software straight from the page provided by the bank we installed this software on the computer.
This software is provided by the bank in order to install on the computer some barriers to prevent attacks.
However this software makes some profound changes in the system.
I do not know how to tell which ones, because I do not have technical knowledge.
When we have the software installed, the site validates the execution of the software and allows the customer access to their checking account.
The question would be only to identify if it is in the scope of nv access to do something so that nvda can work properly on a machine with these protection software installed, or if there is nothing to do besides trying to interact with the bank and complain that the software safety is preventing the nvda from working properly.

**Comment by Adriani90** [comment]
@fernando-jose-silva did you try to run the COM registration fixing tool in NVDA menu / tools?
Also try to maximize the window by pressing windows+up arrow. Sometimes software pieces minimize windows automatically.
Plase give us an update if this is still an issue in NVDA 2019.3 RC1.

**Comment by fernando-jose-silva** [comment]
Yes it is still a problem.
I believe that nvda cannot do anything here.
I am not technical however what seems to happen.
The banks' software identifies when the user enters the bank's website.
From this moment on, the bank's software prevents the user from pressing any character without it being in the fields to enter the account digits.
This causes the nvda's navigation keys to stop working.
Everything will return to normal if you leave the bank's website and open any other page.
The navigation keys will also work again if you restart nvda, although after a few minutes browsing the bank's website the problem will happen again, making it necessary to restart nvda again to use the navigation keys.
I believe that technically nvda can do nothing, if you think only a note in the user guide or other pertinent documents is necessary stating that some internet pages can hijack nvda navigation keys and that the user can try to restart nvda to recover the nvda navigation key functionality.
Brazilian banks were not willing to change the behavior of security tools to be more accessible.

**Comment by Adriani90** [comment]
@fernando-jose-silva in this case I think it is recommended to establish contact between NV Access and the bank and classify NVDA as a secure software which has the permissions to catch information.
You can do that by email for example.
But please also report here any updates in this regard so that we know what to do later on.
Thanks.
cc: @michaelDCurran

**Comment by fernando-jose-silva** [comment]
Thank you

Enviado do meu iPhone

> Em 4 de fev de 2020, à(s) 10:30, Adriani90 <notifications@github.com> escreveu:
> 
> in this case I think it is recommended to establish contact between NV Access and the bank and classify NVDA as a secure software which has the permissions to catch information.
> You can do that by email for example.
> But please also report here any updates in this regard so that we know what to do later on.
> Thanks.
> cc: @michaelDCurran
> 
> —

**Comment by Adriani90** [comment]
I am closing this issue for now, but feel free to comment if there is any relevant update for the community for this issue and we can reopen. Thanks.
